app/uk/gov/hmrc/eoricommoncomponent/frontend/views/business_short_name.scala.html

```diff
@@ -52,9 +52,7 @@
 
             @CSRF.formField
 
-            <legend>
-                <h1 class="govuk-heading-l">@messages(introText)</h1>
-            </legend>
+            <h1 class="govuk-heading-l">@messages(introText)</h1>
```

Comment:
Why don't we change the input text to heading instead?

Reply:
removed Shortname label and associated heading with the input text itself